low-level/files/opt/seagate/sspl/bin/sspl_config

```diff
@@ -139,10 +144,27 @@ config_sspl() {
         sed -i "s|File=.*|File=\/var/log/$PRODUCT_FAMILY/iem/iem_messages\"|g" $RSYSLOG_CONF
     else
         sed -i "s|File=.*|File=\"${LOG_FILE_PATH}\"|g" $RSYSLOG_CONF
-        sed -i "1 s|^.*$|${LOG_FILE_PATH}|g" $DIR_NAME/conf/etc/logrotate.d/iem_messages
+        sed -i "1 s|^.*$|${LOG_FILE_PATH}|g" $SOURCE_CONF_DIR_LOGROTATE/iem_messages
+    fi
+
+    # Trigger logotate hourly basis
+    if [ ! -d "${LOGROTATE_DIR_DEST}" ]; then
+        mkdir -p ${LOGROTATE_DIR_DEST}
```

Comment:
is this communicated to Provisioner ? They should be aware basically

Reply:
Not yet communicated. I saw similar changes done by CSM. PR in review (https://github.com/Seagate/cortx-manager/pull/140)

I also submitted similar changes for HA yesterday.